modules/avm/variables.tf

```diff
@@ -140,3 +140,49 @@ variable "trigger_prefixes" {
   default     = ["modules"]
   description = "List of repository-root-relative paths which should be tracked for changes"
 }
+
```

Comment:
I would suggest changing all these variables into a single object...similar to [this one](https://github.com/schubergphilis/terraform-aws-mcaf-landing-zone/blob/master/variables.tf#L73), for example.

Reply:
Done!